src/irisnet/noncore/ice176.cpp

```diff
@@ -603,7 +603,7 @@ class Ice176::Private : public QObject {
         });
 
         int prflx_priority = c.ic->peerReflexivePriority(lc.iceTransport, lc.path);
-        pair->binding->setPriority(prflx_priority);
+        pair->binding->setPriority(quint32(prflx_priority));
```

Comment:
looks like a bug. priority can't be negative. So type has to be changed and peerReflexivePriority checked to use unsigned values.